sections/faqs/migration-v5.md

```diff
@@ -0,0 +1,22 @@
+## What do I need to do to migrate to v5?
+
+Ready for this?
+
+```sh
+npm install styled-components@beta react@^16.8 react-dom@^16.8 react-is@^16.8
+```
+
+_That's it._ 💥
```

Comment:
We should probably mention that upgrading React (if you're on a lower version) isn't quite that simple?

Reply:
I think that's outside our concern, if you're doing a major React upgrade that's its own can of worms